packages/local_auth/test/local_auth_test.dart

```diff
@@ -149,6 +173,29 @@ void main() {
         );
       });
 
+      test('authenticate with `localizedFallbackTitle` on iOS', () async {
+        final iosAuthMessages =
+            IOSAuthMessages(localizedFallbackTitle: 'Enter PIN');
+        setMockPathProviderPlatform(FakePlatform(operatingSystem: 'ios'));
+        await localAuthentication.authenticate(
+          localizedReason: 'Needs secure',
+          iOSAuthStrings: iosAuthMessages,
+        );
+        expect(
+          log,
+          <Matcher>[
+            isMethodCall('authenticate',
+                arguments: <String, dynamic>{
+                  'localizedReason': 'Needs secure',
+                  'useErrorDialogs': true,
+                  'stickyAuth': false,
+                  'sensitiveTransaction': true,
+                  'biometricOnly': false,
+                }..addAll(iosAuthMessages.args)),
+          ],
+        );
+      });
```

Comment:
You don't need to have this second test; there's nothing different about the code paths at this layer, so this isn't testing anything different than the first one.